scripts/powershell/readme.md

```diff
@@ -9,3 +9,7 @@ List of great repositories that are maintained and owned elsewhere.
 ### Cyber
 
 Cybersecurity powershell things...
+
+### Office-365
+
+Office 365 related scripts...
```

Comment:
@mattco593 , I edited the description to be more general as it won't just have Gov scripts but also Commercial.